src/content/configuration/output.md

```diff
@@ -744,7 +744,7 @@ These options assign the return value of the entry point (e.g. whatever the entr
 If `output.library` is not assigned a non-empty string, the default behavior is that all properties returned by the entry point will be assigned to the object as defined for the particular `output.libraryTarget`, via the following code fragment:
 
 ```javascript
-(function(e, a) { for(var i in a) { e[i] = a[i]; } }(output.libraryTarget, _entry_return_));
+(function(e, a) { Object.assign(e, a); } }(output.libraryTarget, _entry_return_));
 ```
```

Comment:
`Object.assign` had been a more common use, and It's self-evident

Reply:
I'm afraid we can't update it here, those code fragments are generated by webpack as per https://github.com/webpack/webpack/blob/master/lib/library/AssignLibraryPlugin.js#L146, if you think it's a good idea to update it, please create a PR over there.